src/olympia/reviewers/views.py

```diff
@@ -1519,3 +1524,34 @@ def usage_per_version(request, addon):
     )
     patch_cache_control(response, max_age=5 * 60)
     return response
+
+
+@any_reviewer_required
+@reviewer_addon_view_factory
+@non_atomic_requests
+def review_version_redirect(request, addon, version):
+    addon_versions = list(
```

Comment:
Shouldn't need to do cast into a `list()`. Avoiding it means we do once fewer loop on all the versions.

Reply:
https://github.com/mozilla/addons-server/pull/20754/commits/afc49f9ac3ae4435d08cdd4acec0bc5b4e727243 + I changed the list generator into a generator function too, for some possible efficiency with large result sets.